FreeRTOS/Demo/Posix_GCC/main.c

```diff
@@ -446,57 +445,20 @@ void handle_sigint( int signal )
     exit( 2 );
 }
 
+static uint32_t ulEntryTime = 0;
+
 void vTraceTimerReset( void )
 {
-    int xRet;
-    struct timespec ts;
-    ts.tv_sec = 0;
-    ts.tv_nsec = 0;
-
-    xRet = clock_settime( cid, &ts );
-    if( xRet != 0 )
-    {
-        printf( "Could not reset time: %s\n", strerror( errno ) );
-    }
+	ulEntryTime = xTaskGetTickCount();
```

Comment:
hmm tbh this implementation offers very low resolution when compared to what was here previously. of course, you'd need to get it working :)